index.html

```diff
@@ -258,6 +262,17 @@ <h2><dfn>Performance Timeline</dfn></h2>
         map</var>, given <var>entryType</var> as the <a>key</a>.
       </li>
     </ol>
+    <p>The <a
+    href="https://html.spec.whatwg.org/#session-history-document-visibility-change-steps">session
```

Comment:
Does ReSpec not support links to HTML without having to specify the full url? I thought it does now. And even if it does not, please use the multipage version, as this one is pretty unusable.

Reply:
You'd also need to change the HTML comment saying this algorithm is only used by PageVisibility

Reply:
The term is not exported, as far as I can tell, and [ReSpec](https://respec.org/xref/?term=session+history+document+visibility+change+step) cannot find it. (I've tried a few variants, just in case, but can't get a working xref here, or for the "traverse the history" algorithm linked below. Although if we're changing HTML with this to update the comment, then adding 'export' could happen at the same time.)

Reply:
The visibility stuff got moved to HTML now... might want to coordinate with @noamr on changes.

Reply:
`[=page visibility change steps=]` is exported from HTML.
For mature specs (not sure if this change qualified) it's better to put the actual steps linked from HTML rather than use that hook.

Reply:
I'll put together an HTML PR for that then. I'll reach out with questions. Thanks @noamr

Reply:
Resolving; the latest patch doesn't hook into HTML here